mm2src/coins/nft/storage/wasm/wasm_storage.rs

```diff
@@ -877,7 +818,7 @@ impl BlockNumberTable for NftTransferHistoryTable {
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub(crate) struct NftListTable {
     token_address: String,
-    token_id: String,
+    token_id: BeBigUint,
```

Comment:
Can this change cause problems for already existing databases where value was stored as string? If it wasn't actually released, then there is no issue 🙂

Reply:
Nft feature is not in GUI release. ppl who use komodefi repo will have to clean nft db.

Alto I missed the part that `BigUint` is actually a vector so in responses we get `"token_id":[], "token_id":[5], "token_id":[201]`.

I created `serialize_token_id` function for ` #[serde(serialize_with = "serialize_token_id")]` to serialize token_id into String in response and it works for non wasm binary, but I think I also have to add deserialize function bcz in wasm we use deserialization `json::from_value(item.details_json)`. well from value but anyway, may be in the future we decide to use from string deserialization.

Reply:
PS: yeah we `need deserialize_token_id` in `Nft` and `NftTransferHistory`. I will continue tomorrow, if you dont mind,

Reply:
Sure, thanks for your time :slightly_smiling_face:

Reply:
Added `#[serde(serialize_with = "serialize_token_id", deserialize_with = "deserialize_token_id")]` in Nft and NftTransferHistory